release/uname_linux.go

```diff
@@ -1,98 +0,0 @@
-// -*- Mode: Go; indent-tabs-mode: t -*-
-
-/*
- * Copyright (C) 2017 Canonical Ltd
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 3 as
- * published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- *
- */
-
-package release
-
-import (
-	"syscall"
-)
-
-// We have to implement separate functions for the kernel version and the
```

Comment:
It looks like the change removes this comment - should we add it back?

Reply:
it adds back
```diff
+               // c can be uint8 or int8 depending on arch
```
though

Reply:
Yeah parts are still there, the link to the upstream report and the explanation about separate functions sounded worthwhile but I have not looked too deeply so maybe its fine to drop those.

Reply:
yeah, I'll add it back.